### PR TITLE
Feature/589

### PR DESCRIPTION
**Pull request by M1ndForce**
Add correct permissions for methods

#### Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### Checklist
- [x] 🔽 My branch is up-to-date with the "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date front-end and admin part locally, like a charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: modules/currency/currency.permission.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { hasRoles } = require('../../utils/rules');
+const {
+  roles: { SUPERADMIN, ADMIN },
+} = require('../../consts');
+
+const emailChatQuestionMutation = {
+  addCurrency: hasRoles([SUPERADMIN, ADMIN]),
+  deleteCurrency: hasRoles([SUPERADMIN, ADMIN]),
+  updateCurrency: hasRoles([SUPERADMIN, ADMIN]),
```
Comment (AndriyLymych) [inline modules/currency/currency.permission.js:9]: please, add also for method "deleteAllCurrency"

File: modules/email-chat/email-chat.permission.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const {
+  roles: { SUPERADMIN, ADMIN },
+} = require('../../consts');
+
+const currencyMutation = {
+  addEmailQuestion: isAuthorized,
+  makeEmailQuestionsSpam: hasRoles([SUPERADMIN, ADMIN]),
+  answerEmailQuestion: hasRoles([SUPERADMIN, ADMIN]),
+  deleteEmailQuestions: hasRoles([SUPERADMIN, ADMIN]),
+};
```
Comment (AndriyLymych) [inline modules/email-chat/email-chat.permission.js:11]: please,add also for methods getAllEmailQuestions,,getPendingEmailQuestionsCount

File: modules/payment/payment.permission.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { isAuthorized } = require('../../utils/rules');
+
+const paymentQuery = {
+  getPaymentCheckout: isAuthorized,
+};
```
Comment (AndriyLymych) [inline modules/payment/payment.permission.js:5]: it's not need

**Comment by AndriyLymych** [comment]
image
change this permission

**Comment by AndriyLymych** [comment]
![image](https://user-images.githubusercontent.com/48614659/115341304-092a9100-a1b1-11eb-881d-300a2098fd
add for this case

**Comment by AndriyLymych** [comment]
image
change for query and complete for all methods from service

**Comment by AndriyLymych** [comment]
image
change where need and complete for all methods from service

**Comment by AndriyLymych** [comment]
image
change incorrect permissions

File: modules/email-chat/email-chat.permission.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+} = require('../../consts');
+
+const emailChatQuestionQuery = {
+  getAllEmailQuestions: hasRoles([SUPERADMIN, ADMIN]),
+  getPendingEmailQuestionsCount: hasRoles([SUPERADMIN, ADMIN]),
+  getEmailQuestionById: hasRoles([SUPERADMIN, ADMIN]),
+};
+
+const emailChatQuestionMutation = {
+  addEmailQuestion: isAuthorized,
```
Comment (AndriyLymych) [inline modules/email-chat/email-chat.permission.js:13]: it's not necessary

**Review by AndriyLymych — DISMISSED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by yuliiakucher — APPROVED** [review]

**Pull request merged**